Cython/Compiler/Options.py

```diff
@@ -416,6 +418,12 @@ def parse_directive_value(name, value, relaxed_bool=False):
                 name, orig_value))
     elif type is str:
         return str(value)
+    elif name=="fastcall_args":
+        # FIXME - can only take single value argument in a comment
+        if value.lower == "none":
```

Comment:
Shouldn't this be a method call?

Reply:
Yes it should thanks. I suspect this bit is getting rewritten anyway, but I'll be sure to fix it when I do.